packages/errors/src/codes.ts

```diff
@@ -53,6 +48,14 @@ export const SOLANA_ERROR__INVALID_SEEDS_POINT_ON_CURVE = 2800008 as const;
 export const SOLANA_ERROR__COULD_NOT_FIND_VIABLE_PDA_BUMP_SEED = 2800009 as const;
 export const SOLANA_ERROR__PROGRAM_ADDRESS_ENDS_WITH_PDA_MARKER = 2800010 as const;
 
+// Account-related errors.
+// Reserve error codes in the range [3230000-3230999].
+export const SOLANA_ERROR__ACCOUNT_NOT_FOUND = 3230001 as const;
+export const SOLANA_ERROR__MULTIPLE_ACCOUNTS_NOT_FOUND = 32300002 as const;
+export const SOLANA_ERROR__FAILED_TO_DECODE_ACCOUNT = 3230003 as const;
+export const SOLANA_ERROR__EXPECTED_DECODED_ACCOUNT = 3230004 as const;
+export const SOLANA_ERROR__NOT_ALL_ACCOUNTS_DECODED = 3230005 as const;
+
```

Comment:
Should we give these an `ACCOUNT_` prefix? It makes them sound silly, but might be more pattern-respectful.

Reply:
See #2259 😉